pkg/sfu/buffer/buffer.go

```diff
@@ -27,17 +27,17 @@ import (
 )
 
 const (
-	ReportDelta = 1e9
+	ReportDelta = time.Second
 )
 
 type pendingPacket struct {
-	arrivalTime int64
+	arrivalTime time.Time
```

Comment:
Switching to golang `time.Time` so that we have a monotonic clock available for elapsed time calculations.